sacc/windows.py

```diff
@@ -157,8 +177,10 @@ def from_table(cls, table):
         windows: dict
             Dictionary of id -> TopHatWindow instances
         """
-        return {row['id']: cls(row['min'], row['max']) for row in table}
+        return {table.meta['SACCNAME']: cls(table['min'], table['max'])}
 
+    def get_section(self, indices):
+        return TopHatWindow(self.min[indices], self.max[indices])
```

Comment:
can use `return self.__class__(...` here to avoid having to redefine for subclasses.

Reply:
Also this needs a docstring.

Reply:
(waiting on window discussion)